### DISCUSSION TITLE
Crosslink to related collections in data-model settings

### DISCUSSION THREAD

**Discussion by hanneskuettner** [directus/directus#18342] [Feature Requests]
### Summary

Resolve friction in collection editing by adding a link to the related collection in the data model fields page 

![Screenshot 2023-04-27 at 18 53 28](https://user-images.githubusercontent.com/4376726/234935367-664b5a46-dac4-48ee-b75e-8cfef1a695e7.png)

This would make sense for the following local types

- `translations` 
- `m2o` 
- `2m` 
- `o2m` 
- `files`

### Motivation

I often mistakingly click on a collection but meant to edit for example the translations. 
So then I need to go back to the data model list, scroll through my reasonably long models list to find my translation collection and then open it to edit.

This friction could be resolved by including a related collection link to the fields that have related collections.

### Detailed Design

Check local type, add link if applicable

**Comment by rijkvanzanten** [comment]
This feature request has been accepted and is queued to be implemented! You can follow along with the progress here: https://github.com/directus/directus/issues/18344
